### PR TITLE
Added the Gssoc banner

### LINKED ISSUE #654: DOC: Adding GSSOC'24 banner in README.md
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

It is a suggestion to add GSSoc banner in the bottom of the README.md file

### Explain your solution

The GSSoc banner is missing 
Go to Readme.md file
The gssoc'24 banner can be added

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

Closes #654 

### Description: 
I have added the GSSoC'24 banner in the README.md file

- 
-
-

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

> fill the bracket with x. Something like this [x]

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@jfmartinz please review and merge it

File: README.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ ResourceHub is built using the following technologies:
 
 > Website URL: [ResourceHub](https://resourcehubwebsite.vercel.app/)
 
+## GSSoC'24
```
Comment (jfmartinz) [inline README.md:61]:
```suggestion
## Organization
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ ResourceHub is built using the following technologies:
...
+![gssoc](https://miro.medium.com/v2/resize:fit:1100/format:webp/1*fqJaH_oISOR96gLgpJBwWQ.png)
+
```
Comment (jfmartinz) [inline README.md:63]:
```suggestion
ResourceHub is participating in GirlScript Summer of Code (GSSoC), an annual open-source program by the GirlScript Foundation. GSSoC promotes gender diversity and inclusion in tech by enabling participants to contribute to various projects under expert mentorship. We are excited to collaborate with talented individuals and advance our project through this initiative.
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Add it also below technologies table of contents:

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Add the link after the technologies:

**Review by jfmartinz — APPROVED** [review]

**Pull request merged**